dis-segmentation/model/helpers.py

```diff
@@ -0,0 +1,156 @@
+import os
+import sys
+from io import BytesIO
+
+import gdown
+import numpy as np
+import requests
+import torch
+import torch.nn.functional as F
+from torch.autograd import Variable
+from torchvision import transforms
+
+sys.path.append("/app/DIS/IS-Net")
+
+# project imports
+from data_loader_cache import im_preprocess, im_reader, normalize
+from models import *
+
+
+def download_models():
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    # Download official weights
+    if not os.path.exists("./saved_models"):
```

Comment:
nit: better to use `external_data` in the config for something like this.

Reply:
I need to save the model weights inside the cloned repo. `external_data` gets executed before the clone I believe